Backpack/src/main/res/values/backpack.text.size.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Backpack for Android - Skyscanner's Design System
+
+  Copyright 2018 Skyscanner Ltd
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<resources>
+
+  <!-- THIS FILE IS AUTO GENERATED. DO NOT MODIFY! -->
+
+  <dimen name="bpkTextXlSize">24sp</dimen>
```

Comment:
it was a looot simpler to split out the text sizes into a separate file. with xml resources the file doesn't matter to consumers, so it's not a breaking change